python/ray/data/aggregate.py

```diff
@@ -164,8 +178,11 @@ def finalize(a: List[float]):
             init=lambda k: [0, 0, 0],
             accumulate=accumulate,
             merge=merge,
-            finalize=finalize,
-            name=(f"std({str(on)})"))
+            finalize=finalize)
+
+
+Aggregation = Union[str, AggregateFn]
```

Comment:
Could we move this type higher in the file with a forward reference ("AggregateFn")?